client/packages_test.go

```diff
@@ -87,3 +89,45 @@ func (cs *clientSuite) TestClientPackages(c *check.C) {
 		},
 	})
 }
+
+const (
+	appID = "chatroom.ogra"
+)
+
+func (cs *clientSuite) TestClientPackageCallsEndpoint(c *check.C) {
+	_, _ = cs.cli.Package(appID)
```

Comment:
A lot of it boils down to a religious argument I think ;)

Consistency with the rest of the codebase is a greater priority which from what I've seen would mean having a single test that covers the happy path.